types/params.go

```diff
@@ -41,7 +41,7 @@ type ConsensusParams struct {
 	Evidence  EvidenceParams  `json:"evidence"`
 	Validator ValidatorParams `json:"validator"`
 	Version   VersionParams   `json:"version"`
-	Timestamp TimestampParams `json:"timestamp"`
+	Timing    TimingParams    `json:"timing"`
```

Comment:
We should include a note in the changelog and/or upgrade instructions about this change.

Reply:
This is all merging into the pbts branch and changing a field that only exists in that branch. We should add a change log entry / entries for PBTS, but changing something that hasn't landed on master should be fine.

Reply:
That's fair. I just thought it would be easier to add them now, before the eventual merge, so that we wouldn't lose track. But you're right it's not essential that it happen right now.

Reply:
Oh! No problem then, I can add. Just wanted to make sure it was clear what I was doing here.

Reply:
I appreciate that. I do often miss which branch the merge is targeting.